lib/routes/thebanker/index.ts

```diff
@@ -0,0 +1,78 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import { load } from 'cheerio';
+import got from '@/utils/got';
+import { parseDate } from '@/utils/parse-date';
+
+export const route: Route = {
+    path: '/:language?/:channel?/:subChannel?',
```

Comment:
This route is exactly the same as https://github.com/DIYgod/RSSHub/blob/master/lib/routes/rfa/index.ts